examples/echoserver/echoserver.c

```diff
@@ -121,12 +121,14 @@ static int dump_stats(thread_ctx_t* ctx)
 
 #define SSH_TIMEOUT 10
 
-static void callbackReqSuccess(WOLFSSH *ssh, void *buf, word32 sz, void *ctx)
+static int callbackReqSuccess(WOLFSSH *ssh, void *buf, word32 sz, void *ctx)
```

Comment:
I don't see that the client is responding to the server's Global Request. It isn't set up to anywhere, is it?

Reply:
client.c has callbackGlobalReq:371.